addon/components/list-card.js

```diff
@@ -248,3 +287,6 @@ export default Ember.Component.extend({
     }
   }
 });
+
+export default ListCardComponent;
+export { ListCardComponent, QueryToken, QueryFilter, FilterGroup };
```

Comment:
`objects` or `classes` sound good to me....`utils` is generally used in Ember for simple functions, from what I know